ifctrl-wext.c

```diff
@@ -150,8 +150,8 @@ bool ifctrl_iwget_interface_info(const char *interface) {
 }
 
 bool ifctrl_iwget_freqlist(__attribute__((unused)) int phy, struct chan_freq* chan) {
-	conf.num_channels = wext_get_channels(mon, conf.ifname, chan);
-	if (conf.num_channels)
+	int num_channels = wext_get_channels(mon, conf.ifname, chan);
```

Comment:
/Users/jeromy/work/others/horst/ifctrl-wext.c:176:20: error: use of undeclared identifier 'mon'
        if (wext_set_freq(mon, interface, freq))
                          ^
/Users/jeromy/work/others/horst/ifctrl-wext.c:182:31: error: use of undeclared identifier 'mon'
        conf.if_freq = wext_get_freq(mon, interface);
                                     ^
/Users/jeromy/work/others/horst/ifctrl-wext.c:189:40: error: use of undeclared identifier 'mon'
        conf.num_channels = wext_get_channels(mon, conf.ifname, chan);